### PR TITLE
Fixes deepfryers related bugs

### PR DESCRIPTION
**Pull request by Carbonhell**
- bugfix: "Fixes a few bugs related to deepfryer, it won't make you crash anymore."
  - rscdel: "Now items and mobs can get the deep fried prefix only once, to prevent client crashes."
  - rscadd: "You can now remove mobs while they're being fried to save them...or deepfry them again,like the devil you are."

### CODE REVIEW

**Comment by Carbonhell** [comment]
do not merge,gitbash did some weird shit...

**Comment by Carbonhell** [comment]
Should be good to go but if someone could check the code for me it'd be rad. Gitbash added some <<<<<HEAD and other shit which i removed.

**Comment by TDcoolguy300** [comment]
Travis is saying this wont compile, DNM will stay on until this is solved.

File: code/modules/food&drinks/kitchen machinery/deep_fryer.dm [hunk 1]
```diff
@@ -34,12 +38,22 @@
 		return
 	on = TRUE
 	F.loc = src
-	frying = F
```
Comment (TDcoolguy300) [inline code/modules/food&drinks/kitchen machinery/deep_fryer.dm:37]: BYOND is complaining about this being removed, AKA Frying doesn't exist in current context for line 57

File: code/modules/food&drinks/kitchen machinery/deep_fryer.dm [hunk 2]
```diff
@@ -14,12 +14,16 @@
...
-/obj/machinery/deepfryer/examine()
+/obj/machinery/deepfryer/examine(mob/user)
 	..()
-	if(frying)
-		usr << "You can make out [frying] in the oil."
+	if(contents.len)
+		var/list/frying = list2text(contents, ", ")
+		user << "You can make out [frying] in the oil."
+
+/obj/machinery/deepfryer/proc/mob_fry(mob/living/F, mob/user)
```
Comment (TDcoolguy300) [inline code/modules/food&drinks/kitchen machinery/deep_fryer.dm:24]: Double procs

**Comment by TDcoolguy300** [comment]
Just saying, when you made the code THEN synced to the master, your code broke to shit. Went back to the original commit from your code and repasted it into my BYOND, along with the Moving fix i did

**Pull request closed without merging**